docs/tests-ui.rst

```diff
@@ -12,7 +12,7 @@ Installing Dependencies
 
 3. From the `tests/ui/` directory, use NPM or another package manager to install Intern::
 
-    npm install intern
+    npm install intern@2.2.2
```

Comment:
We could alternatively recommend that people run `npm install intern@^2.0.0`. That will install the latest version in the 2.0.0 branch, which should be API compatible. That way we won't need to update the file for every version.